source/includes/administration/_service_connections.md

```diff
@@ -59,3 +59,92 @@ Attributes | &nbsp;
 `name`<br/>*string* | The name of the service connection
 `type`<br/>*string* | The type of the service connection.
 `status`<br/>*Object* | Status of the service connection. Tells you if the service is up.<br/>*includes*: `lastUpdated`, `reachable`
+
+<!-------------------- GET APIINFO -------------------->
+
+### Retrieve API credentials
+
+`GET /services/connections/:id/apiInfo`
+
+```shell
+# Retrieve your API credentials
+curl -X GET \
+  "https://system.cloudmc-dev.cloudops-devteam.com/rest/services/connections/[id]/apiInfo?environmentId=[id]" \
```

Comment:
I think the URL should be using the api, so it has a v1, and we don't use real hostnames.

Something like this might be better: `"https://cloudmc_endpoint/v1/services/connections/fake-id/apiInfo?environmentId=fake-env-id"`

Reply:
Nice catch, will make the change to the hostname